### ISSUE TITLE
APER_OFFSET not preserved after MAKETHIN

### ISSUE THREAD

**Issue by rdemaria** [MethodicalAcceleratorDesign/MAD-X#612]
Feature discussed here #384, implemented here  771b1a20a2d6b5974e63e106b2982807b97e8e58.
However this code 
```
option, warn,info;
system,"ln -fns /afs/cern.ch/eng/lhc/optics/runII/2016 lhc";
call,file="lhc/lhc_as-built.seq";
call,file="lhc/toolkit/macro.madx";
Option, -echo,warn,info;
mbrc.4r1.b2,aper_offset={4,3};
show,mbrc.4r1.b2;
exec,myslice;
show,mbrc.4r1.b2..3;
mbrc.4r1.b2..3,aper_offset={4,3};
```
returns
```
+++ dumping element mbrc.4r1.b2         parent mbrc              
...
parameter: aper_offset   double array: 4.000000e+00 3.000000e+00 
...
 +++ dumping element mbrc.4r1.b2..3      parent multipole         
command: thin_multipole  module: element
parameter: magnet   double value:                  1
parameter: at   double value:              157.9
parameter: l   double value:                  0
parameter: lrad   expression: ( l.mbrc ) / ( 4 )  :: value:             2.3625
double value:             2.3625
parameter: knl   double array: 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 
parameter: kmax   expression: kmax_mbrc_4.5k  :: value:                3.8
double value:                3.8
parameter: kmin   expression: kmin_mbrc_4.5k  :: value:              0.173
double value:              0.173
parameter: calib   expression: kmax_mbrc_4.5k / imax_mbrc_4.5k  :: value:    0.0008636363636
double value:    0.0008636363636
parameter: polarity   double value:                 -1
parameter: mech_sep   double value:             -0.188

+=+=+= fatal: illegal keyword: aper_offset
```

I cannot say if it is a regression...

**Comment by rdemaria** [comment]
The problem seems bigger, because also the other aperture attributes are not created by makethin by default and therefore cannot be modified...

**Comment by HelmutCERN** [comment]
Issue also seen in simple example below:
``` 
option, warn,info;
mq: quadrupole, l=2;
mb: rbend, l=3;
ss: sequence,l=10;
mq, at=1;
mb, at=5;
endsequence;

beam; use,sequence=ss;

select,flag=makethin, class=rbend, slice=2;
select,flag=makethin, class=quadrupole, slice=2;

makethin,sequence=ss;

use,sequence=ss;
mq..1,aper_offset={4,3};
mb..1,aper_offset={4,3};
```

Workaround, save and read the back the sliced sequence
```
save,sequence=ss,file=ss.seq;
call,file=ss.seq;
```

**Comment by tpersson** [comment]
What is the status of this issue at the moment?

**Comment by rdemaria** [comment]
As far as I know Helmut as a new version of makethin that solves this
problem.

**Comment by tpersson** [comment]
This is fixed in #743
